lib/osf-components/addon/components/form-controls/template.hbs

```diff
@@ -37,9 +37,9 @@
         disabled=this.disabled
         onKeyUp=@onKeyUp
     )
-    select=(
+    powerSelect=(
```

Comment:
IMHO, we should just call this `select`. The fact that it's a power-select does not need to be know by the user of form-controls (we could implement our own power-select-like thing, for example).